Source/JavaScriptCore/inspector/protocol/Page.json

```diff
@@ -361,6 +361,56 @@
                 { "name": "frameId", "$ref": "Network.FrameId", "description": "Id of the frame that has been detached." }
             ]
         },
+        {
+            "name": "frameDidStartWebAuthenticationOperation",
+            "description": "Fired when the frame starts a web authentication operation.",
+            "targetTypes": ["page"],
+            "parameters": [
+                {
+                    "name": "ceremonyId",
+                    "type": "string",
+                    "description": "Unique identifier for this authentication ceremony."
+                },
```

Comment:
Style: please inline these as otherwise it's significantly harder to understand the parameter names at a glance
```json
"parameters": [
    { "name": "ceremonyId", "type": "string", "description": "Unique identifier for this authentication ceremony." },
    { "name": "frameId", "$ref": "Network.FrameId" },
    { "name": "request", "type": "object", "description": "The JSON representation of the request for a web authentication operation." },
    { "name": "initiator", "$ref", "Network.Initiator", "optional": true }
]
```